sdk/endpoints/online/managed/managed-identities/online-endpoints-managed-identity-uai.ipynb

```diff
@@ -0,0 +1,535 @@
+    "file_name = \"<FILE_TO_ACCESS>\"\n",
+    "\n",
+    "uai_name = \"<USER_ASSIGNED_IDENTITY_NAME>\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import DefaultAzureCredential, AzureCliCredential\n",
+    "from azure.ai.ml.entities import (\n",
+    "    ManagedOnlineDeployment,\n",
+    "    ManagedOnlineEndpoint,\n",
+    "    Model,\n",
+    "    CodeConfiguration,\n",
+    "    Environment,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "credential = AzureCliCredential()\n",
+    "ml_client = MLClient(credential, subscription_id, resource_group, workspace_name)\n",
+    "\n",
+    "workspace_location = ml_client.workspaces.get(workspace_name).location"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Configure deployment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "define-configuration-2"
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = ManagedOnlineEndpoint(name=endpoint_name, auth_mode=\"key\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "define-configuration-3"
+   },
+   "outputs": [],
+   "source": [
+    "deployment = ManagedOnlineDeployment(\n",
+    "    name=\"blue\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    model=Model(path=\"../../model-1/model/\"),\n",
+    "    code_configuration=CodeConfiguration(\n",
+    "        code=\"../../model-1/onlinescoring/\", scoring_script=\"score_managedidentity.py\"\n",
+    "    ),\n",
+    "    environment=Environment(\n",
+    "        conda_file=\"../../model-1/environment/conda.yml\",\n",
+    "        image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    ),\n",
+    "    instance_type=\"Standard_DS2_v2\",\n",
+    "    instance_count=1,\n",
+    "    environment_variables={\n",
+    "        \"STORAGE_ACCOUNT_NAME\": storage_account_name,\n",
+    "        \"STORAGE_CONTAINER_NAME\": storage_container_name,\n",
+    "        \"FILE_NAME\": file_name,\n",
+    "        # We will update this after creating an identity\n",
+    "        \"UAI_CLIENT_ID\": \"uai_client_id_place_holder\",\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Create the managed identity"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "create-managed-identity-1"
+   },
+   "outputs": [],
+   "source": [
+    "from azure.identity import DefaultAzureCredential\n",
+    "from azure.mgmt.msi import ManagedServiceIdentityClient\n",
+    "from azure.mgmt.msi.models import Identity"
```

Comment:
I've faced an error when loading this module. `ModuleNotFoundError: No module named 'azure.mgmt.msi'`
Can we describe the required libraries at the top of the notebook, so customers can understand what they need to run the notebook?

Reply:
Good idea - added a section